icons/copy-brand-icons.js

```diff
@@ -5,7 +5,7 @@ const fs = require('fs');
 */
 
 const brandIconNames = [
-  'github', 'google', 'linkedin', 'slack', 'instagram', 'facebook', 'twitter', 'twitch', 'youtube', 'discord',
+  'github', 'google', 'linkedin', 'slack', 'instagram', 'facebook', 'twitter', 'twitter-x', 'twitch', 'youtube', 'discord',
```

Comment:
In addition to adding `twitter-x` here, I believe you'll also need to run the icons build process locally to ensure the SVG is copied over (i.e., run `npm run build` in the `icons` directory) to generate the associated JSX/JS files as well. Without doing so, the icon won't exist on the docs site (e.g., it doesn't appear on the deploy preview for this PR) or exposed to consumers of the `@edx/paragon` library.

Docs: https://github.com/openedx/paragon/tree/master/icons#paragon-icons

Reply:
Will do!

Reply:
Okay I ran `npm install` and `npm run build` in the icons directory. I'm not seeing any JSX directly related to twitter-x so if I should do some more research and figure out any steps I missed let me know!

Reply:
I think the problem here might be that `twitter-x` icon is new and we don't have the latest version of `bootstrap-icons` setup in the project, currently the latest version is `v1.11.1` but we have `v1.10.5`, maybe it was only added somewhere in between these releases? Can you try upgrading to `v1.11.1` and running `npm run build` again?

Also, I believe for this you have to run `npm install` in the root directory of the project, not in the icons one, because we [look for bootstrap icons in the root node modules](https://github.com/openedx/paragon/blob/master/icons/copy-brand-icons.js#L17) (that's just a by-product of using npm workspaces 🙂)